Makefile

```diff
@@ -1,41 +1,48 @@
 YARPC=yarpc
 TEST_DIR=tests
 PYTHON?=python3
-PIP?=pip
+PIP3?=pip3
 PYTEST?=pytest
 
 .PHONY: create-env
 create-env:
 	rm -rf env
-	$(PYTHON) -m $(PIP) install --user virtualenv
+	$(PYTHON) -m $(PIP3) install --user virtualenv
 	$(PYTHON) -m venv env
 	source env/bin/activate
 
 .PHONY: install
 install:
-	$(PIP) install --upgrade $(PIP)
-	$(PIP) install -r $(TEST_DIR)/utils/requirements.txt
+	$(PIP3) install --upgrade pip
+	$(PIP3) install .
+	$(PIP3) install -r $(TEST_DIR)/utils/requirements.txt
+	$(PIP3) install pre-commit pytest pytest-cov codecov
 
 .PHONY: test
 test: .cleanCoverage
-	$(PYTEST) $(TEST_DIR)/unit
+	$(PYTEST) $(TEST_DIR)/unit -v
 
 .cleanCoverage:
 	@echo 'cleaning coverage files ...'
-	rm -f .coverage
-	rm -fr htmlcov/
+	rm -rf .coverage htmlcov/
 
 .PHONY: open-report
 open-report: .cleanCoverage
 	$(PYTEST) --cov=$(YARPC) --cov-report=term-missing --cov-report=html
 	open htmlcov/index.html
 
 .PHONY: ci-test
-ci-test:
-	$(PIP) install -r tests/utils/requirements.txt
-	pre-commit run --all-files
-	$(PIP) install .
-	$(PYTEST) --cov --cov-report=xml
+ci-test: lint
+	$(PYTEST) --cov --cov-report=xml -v
+
+.PHONY: lint
+lint:
+	@find . -type d -name 'env' -prune -o -name 'docs' -prune -o -name '*.py' -exec flake8 "{}" +
+	@find . -type d -name 'env' -prune -o -name 'docs' -prune -o -name '*.py' -exec black "{}" +
+	@find . -type d -name 'env' -prune -o -name 'docs' -prune -o -name '*.py' -exec mypy --ignore-missing-imports "{}" +
+	@echo 'flake8....................................................................Passed'
+	@echo 'black.....................................................................Passed'
+	@echo 'mypy......................................................................Passed'
```

Comment:
Copying pre-commit output is a little bit silly. Do we really need that? Also these messages should follow commands, they're referencing

Reply:
I like how aiohttp does this, I'd like to have a separate command for each tool, pass messages aren't needed at all in this case: https://github.com/aio-libs/aiohttp/blob/50d8af73f7c3875d594974f5c13c03f0bd91b2d5/Makefile#L46

Reply:
It is just an output that does not depend on the `pre-commit` but our custom `lint`.

Reply:
Do you want me to remove those messages?

Reply:
Yea, these messages aren't needed

Reply:
Done, I just thought it looks good in Travis